nextcord/ext/commands/core.py

```diff
@@ -423,6 +423,73 @@ def __init__(self, func: Union[
 
         self.checks.extend(self.parent.checks)  # type: ignore
 
+    # Issue #511
+    @property
+    def required_permissions(self) -> Dict[str, bool]:
+        """
```

Comment:
style comment: consistency and PEP257 - brief description on the same line as the opening `"""`